banks/bank_77.asm

```diff
@@ -1,18 +1,409 @@
-Func_077_4000::
-	dr $1dc000, $1dc311
-
-Func_077_4311:
-	ld bc, $80
-.asm_4314
-	ld hl, $0000
-.asm_4317
+Intro_VastFame::
+; Vast Fame logo screen
+	call ClearBGMap0
+
+	xor a
+	ldh [hFFBF], a
+	ldh [hSCX], a
+	ldh [hFFAF], a
+	ldh [hSCY], a
+	ldh [hFFB1], a
+	ld [wdcf3], a
+	ld [wdcf4], a
+	ld [wdcfb], a
+	ld [wdce8], a
+	ld [wdcf6], a
+	ld [wdcfc], a
+	ld [wdcf5], a
+	ld [wdcf7], a
+	ld [wdcf8], a
+	ld [wcd40], a
+	ld [wcd41], a
+
+	ld hl, vBGMap0
+	ld de, VastFameTilemap
+	ld bc, $1412
```

Comment:
It's better to use the `lb` macro here, as the values seem to be independent as shown below.